### PR TITLE
Aggregate javadocs

### PR DESCRIPTION
**Pull request by srnagar**

### CODE REVIEW

File: sdk/parents/azure-client-sdk-parent/pom.xml [hunk 1]
```diff
@@ -866,6 +866,7 @@
                 -snippetpath ${project.basedir}/sdk/storage/azure-storage-queue/src/samples/java
                 -snippetpath ${project.basedir}/sdk/textanalytics/azure-ai-textanalytics/src/samples/java
               </additionalOptions>
+              <aggregate>true</aggregate>
```
Comment (alzimmermsft) [inline sdk/parents/azure-client-sdk-parent/pom.xml:869]: Is this a configuration for the Javadoc plugin or a report configuration? I don't see an option for `aggregate` in the Javadoc plugin documentation

File: pom.xml [hunk 2]
```diff
@@ -7,46 +7,527 @@
...
+          <plugins>
+            <plugin>
+              <groupId>org.apache.maven.plugins</groupId>
+              <artifactId>maven-javadoc-plugin</artifactId>
+              <version>3.1.1</version> <!-- {x-version-update;org.apache.maven.plugins:maven-javadoc-plugin;external_dependency} -->
+              <configuration>
+                <source>1.8</source>
+                <doctitle>Azure SDK for Java Reference Documentation</doctitle>
+                <windowtitle>Azure SDK for Java Reference Documentation</windowtitle>
+                <footer>Visit the &lt;a href="https://docs.microsoft.com/java/azure/"&gt;Azure for Java Developers&lt;/a&gt;site
```
Comment (alzimmermsft) [inline pom.xml:128]:
```suggestion
                <footer>Visit the &lt;a href="https://docs.microsoft.com/java/azure/"&gt;Azure for Java Developers&lt;/a&gt; site
```

File: sdk/parents/azure-client-sdk-parent/pom.xml (hunk 1 above)
Comment (srnagar) [reply sdk/parents/azure-client-sdk-parent/pom.xml]: This change is not required and is reverted.

**Comment by mitchdenny** [comment]
What was the reason for disabling link verification? /cc @sima-zhu

**Comment by srnagar** [comment]
@mitchdenny This CI job is for generating aggregate reports (Javadocs, checkstyle, spotbugs, dependency list) daily. I am not sure why we need link verification in this pipeline. Link verification should have happened in PR builds and daily live test builds.

**Comment by sima-zhu** [comment]
@srnagar The link verification in each PR build only scan against the changed files. CI builds only scan against target the sdk service folder. The link verification in aggregate report is to scan the entire repo to guarantee all links in markdown using the valid ones.

**Comment by srnagar** [comment]
Thanks for clarification @sima-zhu . I have re-enabled link verification.

**Review by alzimmermsft — APPROVED** [review]

**Pull request merged**